exercises/farewell-workflow/practice/src/main/java/farewellworkflow/Starter.java

```diff
@@ -0,0 +1,29 @@
+package farewellworkflow;
+
+import io.temporal.client.WorkflowClient;
+import io.temporal.client.WorkflowOptions;
+import io.temporal.client.WorkflowStub;
+import io.temporal.serviceclient.WorkflowServiceStubs;
+
+public class Starter {
+    public static void main(String[] args) throws Exception {
+
+        WorkflowServiceStubs service = WorkflowServiceStubs.newLocalServiceStubs();
+
+        WorkflowClient client = WorkflowClient.newInstance(service);        
+
+        WorkflowOptions options = WorkflowOptions.newBuilder()
+                    .setWorkflowId("greeting-workflow")
```

Comment:
did you want these to be `farwell`?

Reply:
We seem to have mixed and matched names here. Everything is "Greeting" here except the package name, but that's also how it is in our other 101 courses. I'll confer with @tomwheeler about this